app/controllers/mixins/policy_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module Mixins
+  module PolicyMixin
```

Comment:
Does it make sense to have a mixin for a single method that is being used in a single file?